linera-execution/src/system.rs

```diff
@@ -349,6 +353,8 @@ pub enum SystemExecutionError {
     ArithmeticError(#[from] ArithmeticError),
     #[error(transparent)]
     ViewError(ViewError),
+    #[error(transparent)]
+    BcsError(#[from] bcs::Error),
```

Comment:
The BcsError could be put in the `ViewError`. Non blocker of course.

Reply:
Not sure: I've also heard the argument that we should create errors only in their own module.